Packs/DeCYFIR/Integrations/decyfir/decyfir.py

```diff
@@ -0,0 +1,490 @@
+                                                                              LABEL_SOCIAL_HANDLERS)
+                return_data = return_data + incidents_json_data
+
+            # PHISHING
+            if json_data := json_val.get(VAR_PHISHING):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_DB_WM,
+                                                                              LABEL_PHISHING)
+                return_data = return_data + incidents_json_data
+
+            # ransomware
+            if json_data := json_val.get(VAR_RANSOMWARE):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_DB_WM,
+                                                                              LABEL_RANSOMWARE)
+                return_data = return_data + incidents_json_data
+
+                # Dark web
+            if json_data := json_val.get(VAR_DARK_WEB):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_DB_WM,
+                                                                              LABEL_DARK_WEB)
+                return_data = return_data + incidents_json_data
+
+            # Source Code
+            if json_data := json_val.get(VAR_SOURCE_CODE):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_S_PE,
+                                                                              LABEL_SOURCE_CODE)
+                return_data = return_data + incidents_json_data
+
+            # malicious-mobile-apps
+            if json_data := json_val.get(VAR_MALICIOUS_MOBILE_APPS):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_S_PE,
+                                                                              LABEL_MALICIOUS_MOBILE_APPS)
+                return_data = return_data + incidents_json_data
+
+            # confidential-files
+            if json_data := json_val.get(VAR_CONFIDENTIAL_FILES):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_S_PE,
+                                                                              LABEL_CONFIDENTIAL_FILES)
+                return_data = return_data + incidents_json_data
+
+            # dumps-pii-cii
+            if json_data := json_val.get(VAR_DUMPS_PII_CII):
+                incidents_json_data = self.prepare_incidents_for_digital_risk(json_data, LABEL_DIGITAL_RISK_S_PE,
+                                                                              LABEL_DUMPS_PII_CII)
+                return_data = return_data + incidents_json_data
+
+            return return_data
+        except Exception as e:
+            demisto.error(traceback.format_exc())
+            return_error(str(e))
+            return []
```

Comment:
It will be handled in the main.

```suggestion
           raise DemistoException(str(e))
```